src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -951,6 +951,30 @@ void xclmgmt_connect_notify(struct xclmgmt_dev *lro, bool online)
 	vfree(mb_req);
 }
 
+static int xocl_get_p2p_bar_addr(struct xclmgmt_dev *xdev,
```

Comment:
1. For mailbox protocol, mgmtpf side should not get information from user side. Instead, userpf driver should send p2p bar addr to mgmtpf side.
2. when xocl_p2p_mem_reserve() is called, that means p2p bar is remapped. userpf driver needs to send the new addr to mgmtpf side
3. when mgmtpf driver is up, it notifies the userpf driver (MAILBOX_REQ_MGMT_STATE).  userpf should send the p2p addr to mgmpf

Reply:
Hi Lizhi, I have taken care of your review comments. Please review again and let me know. THanks.